packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -130,23 +135,25 @@ export class MirrorNodeClient {
         return axiosClient;
     }
 
-    constructor(baseUrl: string, logger: Logger, register: Registry, axiosClient?: AxiosInstance) {
-        if (axiosClient !== undefined) {
-            this.baseUrl = '';
-            this.client = axiosClient;
-        } else {
-            if (!baseUrl.match(/^https?:\/\//)) {
-                baseUrl = `https://${baseUrl}`;
-            }
+    constructor(restUrl: string, logger: Logger, register: Registry, restClient?: AxiosInstance, web3Url?: string, web3Client?: AxiosInstance) {
```

Comment:
This works but i wonder in future whether it'd be better to have separate `mirrorNodeRESTClient.ts` and `mirrorNodeWeb3Client.ts` files.
Since they're separate modules and can be changed and deployed separate from the other might be good to have then separate.
No need to address here.